src/backend/data/feed.js

```diff
@@ -37,7 +44,34 @@ class Feed {
    * Returns a Promise.
    */
   save() {
-    addFeed(this);
+    return addFeed(this);
+  }
+
+  /**
+   * Removes current Feed + associated Posts from the databases.
+   * Returns a Promise
+   */
+  async delete() {
+    await removeFeed(this.id);
```

Comment:
Add comments above each line to make it more clear what's happening (eg., `// Remove this feed from redis`)

Reply:
added more comments